apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/project.json

```diff
@@ -8,8 +8,8 @@
 
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
-    "Google.Devtools.AspNet": { "target": "project" },
-    "Google.Devtools.AspNet.Tests": { "target": "project" },
+    "Google.Cloud.Diagnostics.AspNet": { "target": "project" },
+    "Google.Cloud.Diagnostics.AspNet.Tests": { "target": "project" },
     "Google.Devtools.Clouderrorreporting.V1Beta1": "0.3.0",
```

Comment:
This should use Google.Cloud.ErrorReporting.V1Beta1 with target:project, like the ones above. We may need to adjust global.json to make that available, but that's easy enough to do. Basically, every reference to a library within this repo should (currently) use target:project. Once we've gone to GA, we can target specific stable versions instead.